inference-engine/src/transformations/include/transformations/common_optimizations/convert_compression_only_to_legacy.hpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "transformations_visibility.hpp"
+#include "openvino/pass/pass.hpp"
+
+namespace ov {
+namespace pass {
+
+class TRANSFORMATIONS_API ConvertCompressedOnlyToLegacy;
+
+}  // namespace pass
+}  // namespace ov
+
+/**
+ * @ingroup ie_transformation_common_api
+ * @brief ConvertCompressedOnlyToLegacy transformation converts compression only FP16 format to legacy FP16 format.
```

Comment:
"ompression only FP16 format to legacy FP16 format": is it correct?

Reply:
Yes, we call new FP16 format as compression only